/home/mwolff3/.conda/envs/wolff_cancer_ai_env/lib/python3.7/site-packages/autosklearn/data/target_validator.py:188: UserWarning: Fitting transformer with a pandas series which has the dtype category. Inverse transform may not be able preserve dtype when converting to np.ndarray
  "Fitting transformer with a pandas series which"
Traceback (most recent call last):
  File "/home/mwolff3/fail.py", line 27, in <module>
    print(automl.show_models(), indent=4)
TypeError: 'indent' is an invalid keyword argument for print()
